scripts/macos-instructions.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+
+# This script requires yq (https://github.com/mikefarah/yq) which can be installed with this command
+# snap install yq
+
+# This script does two things
+# 1. Modify README.md to include the instructions specified in $INSTRUCTION_FILE
+# 2. Create a github action running all of the code specified in $INSTRUCTION_FILE
+
+# This script must be run in the directory dockstore/dockstore
```

Comment:
Nit: `...in the directory dockstore/dockstore` is a little too specific.

1. I have 2 clones of the the dockstore/dockstore repo (it's easier when switching between radically different branches)
2. Some, at least me before I got overwhelmed with cloned repos, don't bother to create a directory for the dockstore organization.

I would say something like `must be run from the root of the cloned repository` (there is probably better phrasing).

Reply:
**Response to be unable to run script**
I think you have an outdated version of `yq`. When I run the following on my device I get,
```
fhembroff@ll5284-fhembroff:~/Documents$ yq --version
yq (https://github.com/mikefarah/yq/) version 4.26.1
```
and it appears that the macos github action is running `yq (https://github.com/mikefarah/yq/) version 4.26.1`.

Reply:
I changed the message to `# This script must be run in the root of the directory where you cloned the dockstore/dockstore repo`

Reply:
> **Response to be unable to run script** I think you have an outdated version of `yq`. When I run the following on my device I get,
> 
> ```
> fhembroff@ll5284-fhembroff:~/Documents$ yq --version
> yq (https://github.com/mikefarah/yq/) version 4.26.1
> ```
> 
> and it appears that the macos github action is running `yq (https://github.com/mikefarah/yq/) version 4.26.1`.

Can you please update the instructions to include the `yq` version. At some point a couple of years ago, we couldn't upgrade to a newer version, because it broke some sorting we were doing of the openapi.yaml I believe, but I don't think we do that anymore.

Reply:
Added comment stating which version of yq the script was running.

Reply:
I was looking at hard coding the version in, however github actions doesn't seem to have an easy way to alter the version number of pre-installed software.

Reply:
My understanding is that the version of `yq` will stay the same for each version of `macos`, ie. `macos-10`, `macos-11`. However, since we are using `macos-latest` the version of `yq` may change when `macos-latest` changes.